projects/urls.py

```diff
@@ -1,9 +1,23 @@
 from django.conf.urls import url
-
-from projects.views import ProjectListView
+from projects.views import (ProjectListView, ProjectCreate, ProjectDetailView,
+                            ProjectUpdate, ProjectDelete)
 
 
 urlpatterns = [
 
-    url(r'^$', ProjectListView.as_view(), name='list_projects')
+    url(r'^$',
+        ProjectListView.as_view(),
+        name='project_list'),
+    url(r'^details/(?P<pk>\d+)/$',
+        ProjectDetailView.as_view(),
+        name='project_details'),
+    url(r'^create/$',
+        ProjectCreate.as_view(),
+        name='project_create'),
+    url(r'^details/(?P<pk>\d+)/update/$',
+        ProjectUpdate.as_view(),
+        name='project_update'),
+    url(r'^details/(?P<pk>\d+)/delete/$',
```

Comment:
not sure if delete makes sense to be available. At least it should be allowed only for the owner